x-pack/packages/ml/aiops_utils/log_rate_histogram_item.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Log rate histogram item
+ */
+export interface LogRateHistogramItem {
+  /**
+   * Time of bucket
+   */
+  time: number | string;
+  /**
+   * Number of doc count for that time bucket
```

Comment:
Are we able to calculate how the doc count (per bucket) for the deviation compares to the baseline? I end up wanting to know how the counts compare, rather than just a single count number for the baseline / deviation.